src/oc_erchef/apps/chef_index/src/chef_solr.erl

```diff
@@ -70,13 +143,7 @@ search(#chef_solr_query{} = Query) ->
     {ok, Code, _Head, Body} = chef_index_http:request(Url, get, []),
     case Code of
         "200" ->
-            SolrData = jiffy:decode(Body),
-            Response = ej:get({<<"response">>}, SolrData),
-            Start = ej:get({<<"start">>}, Response),
-            NumFound = ej:get({<<"numFound">>}, Response),
-            DocList = ej:get({<<"docs">>}, Response),
-            Ids = [ ej:get({<<"X_CHEF_id_CHEF_X">>}, Doc) || Doc <- DocList ],
-            {ok, Start, NumFound, Ids};
+            handle_successful_search(Body);
```

Comment:
I think this is another case where we can get the provider from the record, and pass it into handle_successful_search instead of needing to look it up in the `handle_successful_search` function.